### ISSUE TITLE
Setting a custom looping animated vector drawable inside of completeIcon

### ISSUE THREAD

**Issue by tutorbear** [cortinico/slidetoact#135]
## What kind of issue is this?

I have an animated vector drawable which is a refresh icon that rotates. I made it using ShapeShifter.desgin . How can I set it as a completed icon and make it loop ? Also, when I click a button, the completed icon will change from the refresh rotation animation to a done tick animation. An example in java would be amazing! TIA

          protected void onCreate(@Nullable Bundle savedInstanceState) {
                  super.onCreate(savedInstanceState);
                  setContentView(R.layout.animation_test_layout);
                  slideToActView = findViewById(R.id.slider_button_new);
                  slideToActView.setCompleteIcon(R.drawable.test_rotate_anim);

          }

       public void stopAnimation(View view) {
           slideToActView.setCompleteIcon(R.drawable.done_tick_anim);
       }

using this code, the rotate animation drawable only rotates once, and the stopAnimation is a button onClick method. When its called, it doesn't start the done tick animation 

 - [ ] UI Bug. Please provide a Screenshot/Video/GIF of what's the problem. If you need help to record a video you can follow [this guide](https://developer.android.com/studio/command-line/adb.html#screenrecord)

 - [ ] Functional Bug. Please describe the scenario of the bug. Furthermore, please spend the time to write a failing test.

 - [ ] Feature Request. Start by telling what's problem you’re trying to solve. A Pull request is welcome as well.

## Details

In case of bug report, please don't forget to include also:
* Library Version.
* Android Version.
* Emulator/Device specs.
* Logs/Crash reports/Stacktraces.
* Example Code/Link to repositories.

**Comment by cortinico** [comment]
> How can I set it as a completed icon and make it loop ?

You should be able to achieve this by updating your object animator of your `AnimatedVectorDrawable` to loop forever:

```
 <objectAnimator
    ...
    android:repeatMode="restart"
    android:repeatCount="infinite"
```

**Comment by tutorbear** [comment]
I have tried that, but it still loops only once.

**Comment by tutorbear** [comment]
The looping works, but is there any way I can set a new complete Icon animation (A done tick animation) when I click a button ? Currently It doesnt show the animation.

**Comment by tutorbear** [comment]
https://media.giphy.com/media/eMDqdhMDsUhGVZtLGn/giphy.gif

This is what the above code is producing now

**Comment by cortinico** [comment]
What you're asking is a duplicate of https://github.com/cortinico/slidetoact/issues/116

This is currently not supported at the moment but, if there is need, I'm happy to discuss a potential implementation for it.
